boards/px4/fmu-v6xrt/default.px4board

```diff
@@ -6,40 +6,63 @@ CONFIG_BOARD_SERIAL_GPS2="/dev/ttyS3"
 CONFIG_BOARD_SERIAL_TEL1="/dev/ttyS2"
 CONFIG_BOARD_SERIAL_TEL2="/dev/ttyS5"
 CONFIG_BOARD_SERIAL_TEL3="/dev/ttyS6"
+CONFIG_BOARD_UAVCAN_TIMER_OVERRIDE=2
```

Comment:
```suggestion
```

Is this one correct?

Reply:
@dagar san
I have included the definition of FMUV6X.
Since it is unknown, should I remove it?